server/app/controllers/dev/seeding/CategoryTranslationFileParser.java

```diff
@@ -58,7 +64,7 @@ public List<CategoryModel> createCategoryModelList() {
   }
 
   private void parseCategoryTranslationFiles() {
-    File directory = new File(CATEGORY_TRANSLATIONS_DIRECTORY);
+    File directory = environment.getFile(CATEGORY_TRANSLATIONS_DIRECTORY);
     if (!directory.exists()) {
```

Comment:
(non-blocking) Recommend you also check `directory.isDirectory()` in case some weirdness happens and you get a file.

Reply:
Great idea.  Done.